dan_layer/engine/src/runtime/impl.rs

```diff
@@ -660,6 +660,17 @@ impl<TTemplateProvider: TemplateProvider<Template = LoadedTemplate>> RuntimeInte
                         true
                     })?;
 
+                    // add event to indicate that there is a change in component
+                    let (template_address, module_name) =
+                        state.current_template().map(|(addr, name)| (*addr, name.to_string()))?;
+                    state.push_event(Event::new(
+                        Some(component_lock.address().clone()),
+                        template_address,
+                        state.transaction_hash(),
+                        "component-updated".to_string(),
```

Comment:
I would use something like "std.component.updated" as topic. The "std." event namespace is reserved for system events, and we also prevent user events from using that prefix. See https://github.com/tari-project/tari-dan/pull/975/files for more info

Reply:
Okay makes sense thanks!
Just pushed.